### ISSUE TITLE
Weird handle movement 

### ISSUE THREAD

**Issue by WaningJupiter** [fontforge/fontforge#5382]
Hello,
I'm using Fontforge [FontForge 2023-01-01] under Mac. 
I have noticed a problem when I move the handles of points (no matter what type of point is), it happens quite often that the handle will not move at the direction lead by the mouse, instead il will move as following an invisible line, I'm not sure it's a bug of Fontforge or I can handle it by reviewing some of the configurations ? Thanks.

https://github.com/fontforge/fontforge/assets/14837045/b246bbe7-5999-4f33-a9f1-81e6f144c1b9

### When reporting a bug/issue:
- [ x] Screenshot
- [ x] The FontForge version and the operating system you're using
- [ x] The behavior you expect to see, and the actual behavior
- [ ] Steps to reproduce the behavior 
- [ ] \(optional) Possible solution/fix/workaround

### When you open an issue for a change/improvement/feature request:
- [ ] A description of the problem you're trying to solve, including _why_ you think this is a problem
- [ ] If the feature changes current behavior, reasons why your solution is better
- [ ] \(optional) Possible solution/fix/workaround

**Comment by iorsh** [comment]
I have the same issue in Linux, and it disappears after I press the Alt-Shift key combination to change the keyboard layout. The strangest bug ever, but I haven't got to investigate it...

**Comment by WaningJupiter** [comment]
Yeah, it's frustrating. What I do is close the glyph and reopen it, 80% of time it works. I will try the Alt+shift, it seems more efficient than the mine. Thanks.
